src/Symfony/Component/Console/Tests/DependencyInjection/AddConsoleCommandPassTest.php

```diff
@@ -76,6 +76,23 @@ public function testProcessRegisterLazyCommands()
         $this->assertSame(array('console.command.symfony_component_console_tests_dependencyinjection_mycommand' => false), $container->getParameter('console.command.ids'));
     }
 
+    /**
+     * @expectedException        \Symfony\Component\DependencyInjection\Exception\InvalidArgumentException
+     * @expectedExceptionMessage  Service "my-command" has multiple "console.command" tags with a different value for the "command" attribute, but the command must have only one name. Use the "alias" attribute instead.
+     */
+    public function testProcessThrowsIfCommandHasMultipleNames()
+    {
+        $container = new ContainerBuilder();
+        $container
+            ->register('my-command', MyCommand::class)
+            ->setPublic(false)
+            ->addTag('console.command', array('command' => 'my:command'))
+            ->addTag('console.command', array('command' => 'other:name'))
+        ;
```

Comment:
> Tag attributes cannot be arrays 

Didnt know that :| And for `name` well.. eh, right. Lets pretend that did not happen :)